### DISCUSSION TITLE
What's the correct BNC adaptor for this radio?

### DISCUSSION THREAD

**Discussion by MicaelJarniac** [ludwich66/Quansheng_UV-K5_Wiki#28] [Q&A] [answered]
I'm struggling with figuring out if it's SMA male/female to BNC male/female.

I believe it should be SMA female, but I'm not sure what the BNC part should be. Male?

I think this information could also be added to the wiki, it sure would have helped me with figuring this out.

**Comment by KvallVall** [comment, marked as answer]
SMA-F to BNC-F.

https://signalstuff.com/product/sma-f-to-bnc-f-adapter/
